src/IndexGrid.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react'
-import {Box, FlexContainer, FlexItem} from '@primer/components'
+import {Box, Flex} from '@primer/components'
 
-const IndexGrid = ({children, ...rest}) => (
-  <Box mx={'auto'} px={5} {...rest} className="container-xl">
-    <FlexContainer {...rest} flexWrap="wrap" alignItems="center" mx={-5}>
-      {children}
-    </FlexContainer>
-  </Box>
-)
+const IndexGrid = ({alignItems, children, flexWrap, ...rest}) => {
+  const flexProps = {alignItems, flexWrap}
+  return (
+    <Box mx={'auto'} px={5} {...rest} className="container-xl">
+      <Flex {...flexProps} mx={-5}>
```

Comment:
It looks like you removed `{...rest}` on `<Flex>` are you sure this doesn't mess up any non-flex props being previously passed down to this component?